projects/igniteui-angular/src/lib/badge/themes/shared/indigo.scss

```
@@ -0,0 +1,17 @@
+@use 'igniteui-theming/sass/bem' as *;
+@use 'igniteui-theming/sass/typography' as *;
+@use 'igniteui-theming/sass/themes/functions' as *;
+@use 'styles/themes/standalone' as *;
+@use '../light/tokens' as *;
+
+@include themed-block(igx-badge, indigo) {
+    @include type-style(button) {
+        margin: 0;
+    }
+
+    igx-icon {
+        $icon-size: rem(12px);
+        --ig-icon-size: #{$icon-size};
+        --igx-icon-size: #{$icon-size};
```

Doesn't that work only by setting the `--size` variable?

```suggestion
        --size: rem(12px);
```